packages/components/src/HttpError/HttpError.scss

```diff
@@ -38,5 +38,12 @@ $tc-http-error-message-color: #888 !default;
 			font-size: 3rem;
 			font-weight: 600;
 		}
+
+		button {
+			padding: 0;
+			font-weight: 600;
```

Comment:
I don't see any spec about font in the guideline. Where does that come from ?

Reply:
The color doesn't seem to be right neither. I think you should see with UX to get those info